go.mod

```diff
@@ -62,6 +62,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
+require github.com/apache/arrow/go/arrow v0.0.0-20211112161151-bc219186db40
```

Comment:
I don't think you wanted to import this (pre-1.0) older version of Arrow. We have it included here because a dependency of Flux needs it, but any new code should use the `v7` import path:
https://github.com/influxdata/flux/blob/6013690a714413da6eb3e44fedfce4d17971d1e3/go.mod#L15